src/main.js

```diff
@@ -0,0 +1,497 @@
+'use strict';
+
+(function () {
+
+const render = (container, template, place) => {
```

Comment:
Добавь `beforeend` как параметр по умолчанию 😃
```suggestion
const render = (container, template, place = `beforeend`) => {
```
`main.js`
```diff
- render(siteHeaderElement, createSiteMenuTemplate(), `beforeend`); 
- render(siteMainElement, createSearchTemplate(), `beforeend`); 
- render(siteMainElement, createFilterTemplate(), `beforeend`); 
- render(siteMainElement, createBoardTemplate(), `beforeend`); 
+ render(siteHeader, createSiteMenuTemplate()); 
+ render(siteMain, createSearchTemplate()); 
+ render(siteMain, createFilterTemplate()); 
+ render(siteMain, createBoardTemplate()`); 
```

Reply:
Убери суффикс `Element`, чтобы не связываться со спорным критерием [`Д 6`](https://up.htmlacademy.ru/ecmascript/9/criteries#d6).

Выражай в названиях признаки только конкретных интерфейсов, которые уникальны на методы и свойства.